src/components/SessionEntry.vue

```diff
@@ -1,178 +0,0 @@
-<template>
```

Comment:
why was this deleted?

Reply:
It's not needed anymore. All the session time computations are happening on the server side.